packages/shared/src/scope.ts

```diff
@@ -0,0 +1,85 @@
+export enum PreservedScope {
+  OpenId = 'openid',
+  OfflineAccess = 'offline_access',
+}
+
+export type UserClaim =
+  | 'name'
+  | 'picture'
+  | 'username'
+  | 'role_names'
+  | 'email'
+  | 'email_verified'
+  | 'phone_number'
+  | 'phone_number_verified'
+  | 'custom_data'
+  | 'identities';
+
+/**
+ * Scopes for ID Token and Userinfo Endpoint.
+ */
+export enum UserScope {
```

Comment:
since @IceHe  asked. If we define the enum here, should we make @logto/shared public and add it as the dependency of js SDKs?  If not, should we just keep js SDK user scopes off track?

Reply:
it's already public. i'm ok with both